### ISSUE TITLE
Undesired interactivity

### ISSUE THREAD

**Issue by mih** [datalad/datalad-container#5]
I just wanted to know whether there are containers present....

```
(datalad3-dev) mih@meiner /tmp/func (git)-[master] % datalad containers-list
Container location
path within the dataset where to store containers [.datalad/environments]: 
```

After I hit return, I am left with a dirty dataset:

```diff
--- a/.datalad/config
+++ b/.datalad/config
@@ -2,3 +2,5 @@
        id = e1b670f4-373b-11e8-9540-f0d5bf7b5561
 [datalad "metadata"]
        nativetype = dicom
+[datalad "containers"]
+       location = .datalad/environments
```

**Comment by mih** [comment]
Same was true for `add` and was removed in #7

**Comment by mih** [comment]
Fixed in #9
